sssom/cliques.py

```diff
@@ -82,8 +82,8 @@ def split_into_cliques(msdf: MappingSetDataFrame):
     return newdocs
 
 
-def invert_dict(d: dict) -> dict:
-    invdict = {}
+def invert_dict(d: Dict[str, str]) -> Dict[str, str]:
+    invdict = {}  # type: Dict[Any, Any]
```

Comment:
use syntax-based type annotation

```suggestion
     invdict: Dict[Any, Any] = {}
```

We should also be able to do better than Any, Any

Reply:
maybe Dict[str, str] ? But I do not know the general data format of objects expected for this function. Thoughts @matentzn ?

Reply:
@hrshdhgd not knowing is just the first step on the journey to knowing! The way I figure this stuff out is to look through the codebase and see where the function is used (you can basically act like a type checker yourself) and see what goes in then what comes out.

Reply:
No ideas about the clique.py code. Ask Chris, but only if these are PR breakers.

Reply:
Figured this out!